executor/modes/local/progress_reporter.py

```
@@ -0,0 +1,224 @@
+                    "task_title": self.task_title,
+                    "subtask_title": self.subtask_title,
+                    "progress": progress,
+                    "status": status,
+                    "message": message,
+                    "result": result,
+                },
+            )
+            logger.debug(
+                f"Progress reported: task_id={self.task_id}, progress={progress}%, status={status}"
+            )
+        except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)